packages/data-context/src/DataContext.ts

```diff
@@ -220,11 +219,6 @@ export class DataContext {
     return new ProjectDataSource(this)
   }
 
-  @cached
```

Comment:
Good bye remotePolling ⚰️ 

Seems our polling / subscriptions story is getting more well defined, any need to update any docs anywhere? We might want to do an update of https://github.com/cypress-io/cypress/blob/develop/guides/graphql-subscriptions.md at some point to capture the ins-and-outs of the whole polling situation, it's not really documented and not very standard - I think it'd be a bit challenging to onboard someone, even with GraphQL experience, to our current App <> Cloud setup.

Not going to block here, but might be a nice thing to do if we have some free time between now and Q3 getting under way.